src/config.ts

```diff
@@ -50,6 +50,9 @@ export default {
       'bye', 'seeya', 'goodbye', 'good bye', 'thanks', 'whatsup',
       'look', 'you',
     ],
+    punctuation: [
```

Comment:
As we nam ed the other one `prepend` I would name this one `append` to be consistent.

I was just thinking, does `append` need a space? So if someone missed off the punctuation, for example

```
hey guys look what I did
```

and potentially a blank one? for just `hey guys`?

Reply:
I added the space to fix this issue: https://github.com/EddieJaoudeCommunity/EddieBot/issues/261 Do you have any suggestions?
I named it punctuation so it's easy to see grammar. I can change it no problem.

Reply:
I personally would prefer `append`